app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,6 @@
     <color name="teal_700">#FF018786</color>
     <color name="black">#FF000000</color>
     <color name="white">#FFFFFFFF</color>
+    <color name="text_main_activity_title_cream">#a9bd97</color>
+    <color name="button_main_activity_cream">#a9bd97</color>
 </resources>
```

Comment:
add an empty line at the end of the file to avoid this error

Reply:
done